pkg/cache/cache.go

```diff
@@ -94,12 +123,13 @@ const (
 // ClusterQueue is the internal implementation of kueue.ClusterQueue that
 // holds admitted workloads.
 type ClusterQueue struct {
-	Name                 string
-	Cohort               *Cohort
-	RequestableResources map[corev1.ResourceName]*Resource
-	UsedResources        ResourceQuantities
-	Workloads            map[string]*workload.Info
-	NamespaceSelector    labels.Selector
+	Name                      string
+	Cohort                    *Cohort
+	RequestableResources      map[corev1.ResourceName]*Resource
+	UsedResources             ResourceQuantities
+	Workloads                 map[string]*workload.Info
+	WorkloadsWithoutPodsReady sets.String
```

Comment:
I am trying to find a better name, this is hard to parse, may be `NotReadyWorkloads` or `WorkloadsNotReady` to keep Workloads are the prefix?

Reply:
I like `WorkloadsNotReady`, renamed